netbox/templates/wireless/wirelesslink.html

```diff
@@ -1,30 +1,31 @@
 {% extends 'generic/object.html' %}
 {% load helpers %}
 {% load plugins %}
+{% load i18n %}
 
 {% block content %}
   <div class="row">
     <div class="col col-md-6">
       <div class="card">
-        <h5 class="card-header">Interface A</h5>
+        <h5 class="card-header">{% trans "Interface A" %}</h5>
```

Comment:
Where we use A/B as arbitrary designations, maybe we should exclude them from the translated text?